kubectl-plugin/kubectl-waiops

```diff
@@ -16,9 +42,228 @@ fi
+
+        echo "______________________________________________________________"
+        echo "AutomationUIConfig, AutomationBase, Cartridge, and CartridgeRequirements instances:" && echo ""
+        oc get automationuiconfig,automationbase,cartridge,cartridgerequirements -A -o custom-columns='KIND:.kind,NAMESPACE:.metadata.namespace,NAME:.metadata.name,VERSION:status.versions.reconciled,STATUS:.status.conditions[?(@.type=="Ready")].status'
+        
+        echo "______________________________________________________________"
+        echo "Kong instances:" && echo ""
+        oc get kong -A -o custom-columns="KIND:kind,NAMESPACE:metadata.namespace,NAME:metadata.name"         
+        
+        echo "______________________________________________________________"
+        echo "ZenService instances:" && echo ""
+        oc get zenservice -A -o custom-columns='KIND:.kind,NAME:.metadata.name,NAMESPACE:.metadata.namespace,VERSION:status.currentVersion,STATUS:.status.zenStatus,PROGRESS:.status.Progress,MESSAGE:.status.ProgressMessage'
+        
+        echo "______________________________________________________________"
+        echo "CSVs from $PROJECT_NAMESPACE namespace:" && echo ""
+        oc get csvs -n $PROJECT_NAMESPACE 
+    
+        echo "______________________________________________________________"
+        echo "CSVs from ibm-common-services namespace:" && echo ""
+        oc get csvs -n ibm-common-services
+
+        echo "______________________________________________________________"
+        echo "Subscriptions from $PROJECT_NAMESPACE namespace:" && echo ""
+        oc get subscriptions -n $PROJECT_NAMESPACE 
+
+        echo "______________________________________________________________"
+        echo "Subscriptions from ibm-common-services namespace:" && echo ""
+        oc get subscriptions -n ibm-common-services
+        
+        echo "______________________________________________________________"
+        echo "OperandRequest instances:" && echo ""
+        oc get operandrequests -A -o custom-columns="NAMESPACE:metadata.namespace,NAME:metadata.name,PHASE:status.phase,CREATED AT:metadata.creationTimestamp"
+        
+        echo "______________________________________________________________"
+        echo "ODLM pod current status:" && echo ""
+        oc get pod -n ibm-common-services | grep operand-deployment-lifecycle-manager
+
+        echo "______________________________________________________________"
+        echo "Orchestrator pod current status:" && echo ""
+        oc get pod -n $PROJECT_NAMESPACE | grep ibm-aiops-orchestrator-controller-manager
+        
+        echo ""
+    else
+        echo "Sorry, this tool only supports CP4WAIOps v3.3 at this time. 
+
+You currently have your CP4WAIOps version in this script set to \"$CP4WAIOPS_VERSION\". 
+
+If this is incorrect, please update the \`CP4WAIOPS_VERSION\` variable
+in this script to \"3.3\" and run \`oc waiops status\` again.
+"
+    fi
+    exit 0
+fi
```

Comment:
IFF this were required, we should do the version check and validation ourselves against the deployed instance.

I think we need to reconsider what we are adding here limitation wise

Reply:
Removed version from script